core-backend/wasmi/src/funcs_tree.rs

```diff
@@ -46,7 +46,7 @@ macro_rules! wrap_common_func_internal_ret {
             let func = move |caller: Caller<'_, HostState<Ext>>, $($arg_name,)*| -> Result<(_, ), Trap>
             {
                 let mut ctx = CallerWrap::prepare(caller, forbidden, memory)?;
-                $func(&mut ctx, $($arg_name,)*).map(|ret| (ret,))
+                $func(&mut ctx, 0, $($arg_name,)*).map(|(_, r)| (r,))
```

Comment:
I think it's better to add comment that 0 is unused gas parameter especially in wasmi backend